build.sbt

```diff
@@ -189,7 +189,7 @@ lazy val midas      = ProjectRef(firesimDir, "midas")
 lazy val firesimLib = ProjectRef(firesimDir, "firesimLib")
 
 lazy val firechip = (project in file("generators/firechip"))
-  .dependsOn(boom, icenet, testchipip, sifive_blocks, sifive_cache, sha3, utilities, tracegen, midasTargetUtils, midas, firesimLib % "test->test;compile->compile")
+  .dependsOn(boom, icenet, testchipip, example, sifive_blocks, sifive_cache, sha3, utilities, tracegen, midasTargetUtils, midas, firesimLib % "test->test;compile->compile")
```

Comment:
Im fine with putting `example` here... however this means `firechip` will compile `hwacha`. Is that fine for FireSim folks? If we do add `hwacha` builds... it would only build once every so often so its not that bad.

Reply:
Also, we could remove the `boom`, `sifive_blocks`, `sifive_cache`, `utilities`, `sha3` to make this cleaner

Reply:
Adding the dependency example is fine; incremental recompilation works pretty well.

Reply:
Adding the dependency on example is fine; incremental recompilation works pretty well.

Reply:
```suggestion
  .dependsOn(example, icenet, testchipip, tracegen, midasTargetUtils, midas, firesimLib % "test->test;compile->compile")
```

Reply:
There were already other redundant, transitive dependencies in the `build.sbt`. I leave finding them as an exercise to the reader.